Packs/MS-ISAC/Integrations/MSISAC/MSISAC.yml

```
@@ -0,0 +1,194 @@
+      type: string
+    - contextPath: MSISAC.Event.Stream.bytes_toclient
+      description: The size of the data sent to the client
+      type: number
+    - contextPath: MSISAC.Event.Stream.src_port
+      description: The source port for the traffic
+      type: number
+    - contextPath: MSISAC.Event.Stream.dest_port
+      description: The destination port for the traffic
+      type: string
+    - contextPath: MSISAC.Event.Stream.event_type
+      description: The type of event submitted from MS-ISAC
+  - arguments:
+    - defaultValue: "1"
+      description: The number of days worth of events to return. Must be one or greater
+      name: days
+      required: true
+    - description: If you want to search the list of events for a specific event,
+        specify this optional command to return just those results
+      name: event_id
+    description: Retrieves a list of MS-ISAC events for a given number of days (one
+      or greater)
+    name: msisac-retrieve-events
+    outputs:
+    - contextPath: MSISAC.RetrievedEvents.event_id
+      description: ID for the retrieved MS-ISAC event
+      type: number
+    - contextPath: MSISAC.RetrievedEvents.stime
+      description: The time that the traffic started
+      type: date
+    - contextPath: MSISAC.RetrievedEvents.sourceip
+      description: The IP that originated the traffic
+      type: string
+    - contextPath: MSISAC.RetrievedEvents.analyzed_ts
+      description: The time that this traffic was analyzed by MS-ISAC
+      type: date
+    - contextPath: MSISAC.RetrievedEvents.logical_sensor_id
+      description: The ID for the sensor that triggered the event
+      type: string
+    - contextPath: MSISAC.RetrievedEvents.ticket_id
+      description: String representation of event_id
+      type: string
+    - contextPath: MSISAC.RetrievedEvents.queue
+      description: The group that originated the event
+      type: string
+    - contextPath: MSISAC.RetrievedEvents.status
+      description: The current state of the event
+      type: string
+    - contextPath: MSISAC.RetrievedEvents.previous_escalations
+      description: How many times this alert has been escalated
+      type: string
+    - contextPath: MSISAC.RetrievedEvents.last_stime
+      description: The last time that this traffic was observed (stop time)
+      type: date
+    - contextPath: MSISAC.RetrievedEvents.sensor
+      description: The hostname of the sensor that triggered the event
+      type: string
+    - contextPath: MSISAC.RetrievedEvents.analysis
+      description: The analysis provided by MS-ISAC
+      type: string
+    - contextPath: MSISAC.RetrievedEvents.description
+      description: The description of the event
+      type: string
+    - contextPath: MSISAC.RetrievedEvents.severity
+      description: The severity assigned to the MS-ISAC alert
+      type: string
+  dockerimage: demisto/python3:3.9.8.24399
+  runonce: false
+  script: ''
+  subtype: python3
+  type: python
```

Please add the `fromversion` field to determine what is the minimum version of Cortex XSOAR that the integration works with. for example:

```suggestion
  type: python
fromversion: 5.5.0
```